src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -156,17 +156,21 @@ class SoftmaxRegression
    * Train the softmax regression with the given training data.
    *
    * @tparam OptimizerType Desired optimizer type.
+   * @tparam CallbackTypes Types of Callback Functions.
    * @param data Input data with each column as one example.
    * @param labels Labels associated with the feature data.
    * @param numClasses Number of classes for classification.
    * @param optimizer Desired optimizer.
    * @return Objective value of the final point.
+   * @param callbacks Callback Functions.
```

Comment:
Can you point to the ensmallen documentation, see https://github.com/mlpack/mlpack/pull/2076 for more details.